src/docs/docsite.json

```diff
@@ -6,6 +6,7 @@
     "3rdparty_py": "dist/markdown/html/examples/src/python/example/3rdparty_py.html",
     "announce_201409": "dist/markdown/html/src/docs/announce_201409.html",
     "coursier_migration": "dist/markdown/html/src/docs/blog/coursier_migration.html",
+    "thoughts_on_linting": "dist/markdown/html/src/docs/blog/thoughts_on_linting.html",
```

Comment:
So, while I definitely appreciate this, I don't think we want this published as a blogpost, because it talks about avoiding pants for a usecase that we actually support.

I would suggest either moving this content to a ticket/issue that we can resolve when/if the feedback is incorporated into pants.

Reply:
I'll do it, because I get your point that pointing shortcomings of our own tool on its website may feel a bit strange; but an issue also doesn't feel like the best place to put this:
I would expect an issue to address one specific problem that can be addressed with a PR. For instance,
* pants overhead for no-op is too high
or
* pants does too much work when asked to lint a single file
or
* port gofmt to v2.

This content is more of a high-level discussion on architecting v2 linters, which feels wishy-washy for an issue; and more of a bloggy thing imo.

Just sharing some thoughts. I'll go ahead with an issue as I don't have a better suggestion.